src/lib/chains/evm/evmErrors.ts

```diff
@@ -0,0 +1,27 @@
+export enum EvmError {
```

Comment:
Please do not use all caps in string fields.  Makes it seem like we are yelling at the developers.  All caps is ok for variable constant names, not in strings,

Reply:
@Elaixzar all caps are indication of global constants